src/sass/patterns/_form-elements.scss

```diff
@@ -1,16 +1,47 @@
 // Form Elements
 // =============
 
-.vertical-separator {
-  border-right: 1px solid $color-border-separator-alt;
+.form-grid {
+  display: grid;
+  grid-template-columns: 1fr 25%;
+  flex-grow: 1;
+  width: calc(100% - 2.25rem);
+}
+
+[data-form='task-capture'] {
+  display: grid;
   height: 100%;
-  width: 1px;
 
-  .hide-separator + & {
-    display: none;
+  // using slds class since we can't access inner checkbox + label markup
+  .slds-checkbox__label {
+    &:hover {
+      .slds-form-element__label {
+        color: $brand-text-link-active;
+      }
+
+      .slds-checkbox_faux,
+      .slds-checkbox--faux {
+        border-color: $brand-text-link-active;
+      }
+    }
   }
 
-  .show-separator + & {
-    display: inline;
+  // adding input focus since keyboard nav doesn't focus on spans
+  [type='checkbox'] {
+    &:focus {
+      + .slds-checkbox__label {
+        .slds-form-element__label {
+          color: $brand-text-link-active;
+        }
+      }
+    }
   }
 }
+
+.selectall-header {
+  margin-left: 18px; // 18px chevron icon placeholder
```

Comment:
I think this is the best place to capture any magic numbers. If you get this row aligned with the others, and using the same grid - things should line up internally.

Reply:
the newest set of changes resolves this